taskfiles/lint.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+
+tasks:
+  cpp:
+    desc: lint cpp code using clang-format
+    cmds:
+    - |
+      PATH="{{.LLVM_INSTALL_PATH}}/bin":$PATH
+      find . -regex '.*\.\(cpp\|h\|hpp\|cc\|proto\|java\)' | xargs -n1 clang-format -i -style=file -fallback-style=none
```

Comment:
`(lang1 | lang2 | lang3 ...)$`  - add $ for end of line/name.

Reply:
otherwise lgtm.

Reply:
sounds good. just pushed the update